### DISCUSSION TITLE
求助如何添加可视面板

### DISCUSSION THREAD

**Discussion by inzaghiaimar** [kkkgo/PaoPaoDNS#207] [问答和帮助]
如题了，请问下各位能否添加个可视面板，这样就可以不用 adg 来看了，可以少个程序更加节省资源

**Comment by biggbuddy** [comment]
不太明白用了paopaodns后为什么还加一层adg？是为了去广告吗？

**Reply by xingshu1990** [reply]
有些是为了直观的看统计数据，有些是为了去广告（虽然这个去广告的效果不是很好。）

**Reply by biggbuddy** [reply]
> 有些是为了直观的看统计数据，有些是为了去广告（虽然这个去广告的效果不是很好。）

一般会关注什么类型的统计数据?

**Reply by xingshu1990** [reply]
不是我想要这个功能，但是楼主要面板UI，大差不差包括但不限于下面的内容
1.查看运行日志，能导出到本地。
2.查看软件自动下载了哪些网址的什么文件，能预览文件内容（为了避免浏览器卡死，预览部分内容）,直接在界面添加、删除、修改内容，类似passwall的直连、proxy、ad界面。
3.类似adguardhome那样，统计DNS查询次数、无法通信的网址信息，每条网址DNS查询的时间，有哪些客户端请求，请求次数分别是多少。

**Reply by inzaghiaimar** [reply]
确实，主要有的功能就是查看日志，统计DNS查询次数、无法通信的网址信息，每条网址DNS查询的时间，有哪些客户端请求，请求次数分别是多少，以及错误查询

**Comment by inzaghiaimar** [comment]
@biggbuddy  我不想用 adg 的面板，但是数据看不到，所以求助一个可用面板

**Reply by xingshu1990** [reply]
> @biggbuddy 我不想用 adg 的面板，但是数据看不到，所以求助一个可用面板


目前来看，作者估计没开发这个面板。
问了一下chatgpt，类似的clash的面板是用什么软件开发，得到的答复是：Electron。
只能是期待哪个大佬开发面板了。
